website/build/react-square-payment-form/docs/SquarePaymentForm.html

```diff
@@ -61,7 +61,7 @@
 <ul>
 <li><strong>applicationId</strong>: This can be found in your <a href="https://connect.squareup.com/apps">Square Developer Dashboard</a>
 for the current Square app you're developing</li>
-<li><strong>locationID</strong>: You can retrieve this from the <a href="https://docs.connect.squareupstaging.com/api/connect/v2#navsection-locations">Square Connect v2 Locations API</a>;
+<li><strong>locationID</strong>: You can retrieve this from the <a href="https://docs.connect.squareup.com/api/connect/v2#navsection-locations">Square Connect v2 Locations API</a>;
```

Comment:
The `website/build` directory is generated when we publish documentation changes to the github pages. So you only need to update copy in the `docs` directory, and we can delete the manual changes to `website/build` files.

I'll try to figure out a better way to convey this and the component documentation changes.